<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lecturer_Attendance_Scanner</class>
 <widget class="QMainWindow" name="Lecturer_Attendance_Scanner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="headerGroup">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>360</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
	border: 0px;
	border-radius: 0px;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="headerHud">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>360</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/res/assets/header_short.png</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="backButton">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>15</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 0px;
	border-radius: 0px;
	background: none;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/res/assets/back.png</normaloff>:/res/assets/back.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="headerSubjectCodeLabel">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>15</y>
       <width>310</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins</family>
       <pointsize>-1</pointsize>
       <italic>false</italic>
       <fontweight>DemiBold</fontweight>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: #FFF;
	text-align: center;
	font-family: Poppins;
	font-size: 24px;
	font-style: normal;
	font-weight: 600;
	line-height: normal;
}</string>
     </property>
     <property name="text">
      <string>SC&lt;?&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="headerSubjectDescLabel">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>45</y>
       <width>310</width>
       <height>15</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: #FFFFFF;
	text-align: center;
	font-family: Poppins;
	font-size: 9px;
	font-style: normal;
	font-weight: 600;
	line-height: normal;
}</string>
     </property>
     <property name="text">
      <string>SD&lt;?&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="headerProgramLabel">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>65</y>
       <width>310</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: #FFFFFF;
	text-align: center;
	font-family: Poppins;
	font-size: 12px;
	font-style: normal;
	font-weight: 400;
	line-height: normal;
}</string>
     </property>
     <property name="text">
      <string>P&lt;?&gt; YS&lt;?&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="footerGroup">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>510</y>
      <width>360</width>
      <height>130</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
	border: 0px;
	border-radius: 0px;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="footerHud">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>340</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/res/assets/footer.png</pixmap>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="homeButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 0px;
	border-radius: 0px;
	background: none;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/res/assets/home_unselected.png</normaloff>:/res/assets/home_unselected.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="homeLabel">
     <property name="geometry">
      <rect>
       <x>105</x>
       <y>105</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: #FFFFFF;
	text-align: center;
	font-family: Poppins;
	font-size: 6px;
	font-style: normal;
	font-weight: 500;
	line-height: normal;
}</string>
     </property>
     <property name="text">
      <string>HOME</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="classListButton">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>80</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 0px;
	border-radius: 0px;
	background: none;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/res/assets/class_list_selected.png</normaloff>:/res/assets/class_list_selected.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="classListLabel">
     <property name="geometry">
      <rect>
       <x>195</x>
       <y>105</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	color: #15CAE3;
	text-align: center;
	font-family: Poppins;
	font-size: 6px;
	font-style: normal;
	font-weight: 500;
	line-height: normal;
}</string>
     </property>
     <property name="text">
      <string>CLASS LIST</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="confirmButton">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>25</y>
       <width>210</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border: 0px;
	border-radius: 0px;
	background: none;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/res/assets/confirm.png</normaloff>:/res/assets/confirm.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>300</width>
       <height>50</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="scannerGroup">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>360</width>
      <height>430</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
	border: 0px;
	border-radius: 0px;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="cameraView" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>360</width>
       <height>360</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget {
	background: #000000;
}</string>
     </property>
     <widget class="QLabel" name="topLeftCorner">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>15</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/res/assets/camera_corner_topleft.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="topRightCorner">
      <property name="geometry">
       <rect>
        <x>285</x>
        <y>15</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/res/assets/camera_corner_topright.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="bottomLeftCorner">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>285</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/res/assets/camera_corner_bottomleft.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="bottomRightCorner">
      <property name="geometry">
       <rect>
        <x>285</x>
        <y>285</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/res/assets/camera_corner_bottomright.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="dataGroup">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>370</y>
       <width>310</width>
       <height>60</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="dataHud">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>310</width>
        <height>60</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/res/assets/data_display.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="avatarLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/res/assets/cict_avatar.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="dataNameLabel">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>8</y>
        <width>180</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	color: #FFFFFF;
	font-family: Poppins;
	font-size: 12px;
	font-style: normal;
	font-weight: 600;
	line-height: normal;
}</string>
      </property>
      <property name="text">
       <string>LN&lt;?&gt;, FN&lt;?&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="dataProgramLabel">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>180</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	color: #FFFFFF;
	font-family: Poppins;
	font-size: 9px;
	font-style: normal;
	font-weight: 400;
	line-height: normal;
}</string>
      </property>
      <property name="text">
       <string>C&lt;?&gt; - P&lt;?&gt; YS&lt;?&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="dataStudentIdLabel">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>35</y>
        <width>180</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	color: #21C5DF;
	font-family: Poppins;
	font-size: 9px;
	font-style: normal;
	font-weight: 400;
	line-height: normal;
}</string>
      </property>
      <property name="text">
       <string>20**M****</string>
      </property>
     </widget>
    </widget>
   </widget>
   <zorder>footerGroup</zorder>
   <zorder>headerGroup</zorder>
   <zorder>scannerGroup</zorder>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
